packages/core/src/helpers/space/space.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * Returns the `px` size of a multiplier of the base space.
  *
- * Only integers and (1.5, 0.5, 0.25, 0.125) are supported.
+ * @note Only integers and (1.5, 0.5, 0.25, 0.125) are supported.
```

Comment:
Why `@note`? I feel like such should be introduced per whole codebase with separate PR...

Reply:
Fair enough, removing.